tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py

```diff
@@ -197,3 +198,94 @@ def test004_account_add_delete_user(self):
                          [user['userGroupId'] for user in account['acl']])
 
         self.log('%s ENDED' % self._testID)
+
+    def test005_get_account_info(self):
+        """ ZRT-OVC-022
+        *Test case for getting account info*
+
+        **Test Scenario:**
+
+        #. Create an account (A1).
+        #. Get A1 and check its info.
+        """
+        self.log('%s STARTED' % self._testID)
+
+        self.log('Create an account (A1)')
+        openvcloud_ser_name = self.random_string()
```

Comment:
what's ser meaning , I think need more clear name

Reply:
means service .. i changed to be more clear